### DISCUSSION TITLE
update on AI's character database

### DISCUSSION THREAD

**Discussion by syzku** [etherealxx/etherportal-webui-colab#10] [Ideas]
as more and more anime styled games and anime series emerging and new characters coming up in already existing games, is it possible to add them up in the AI database so the AI can recognize them? since there are already characters that are not recognized by the AI now, so it takes more effort for the user to make the AI replicate the character when generating images.

**Comment by etherealxx** [comment]
@syzku there is something called LoRA. search for that on google, there's so much explanation. It allows the AI to recognize more character/artstyle.
